docs/development/submitting-patches.rst

```diff
@@ -57,6 +57,61 @@ Important information should be documented with prose in the ``docs`` section.
 To ensure it builds and passes `doc8`_ style checks you can run
 ``make docs`` and ``make lint`` respectively.
 
+Keeping your local branch updated
+---------------------------------
+
+As you work, it is helpful to keep your local branch up-to-date with the PRs
+that are merged into the main Warehouse repository. Most people working on
+the project use this workflow.
+
+The following documentation assumes that you have git configured like this:
+
+.. code-block:: console
+
+  $ git remote -v
+  origin  https://github.com/USERNAME/warehouse.git (fetch)
+  origin  https://github.com/USERNAME/warehouse.git (push)
+  upstream  https://github.com/pypa/warehouse.git (fetch)
+  upstream  https://github.com/pypa/warehouse.git (push)
+
+
+In the example above, USERNAME is your username on GitHub.
```

Comment:
Please put backticks or similar around code-type words like USERNAME in this document.